### PR TITLE
Combine web trackers sections

### PR DESCRIPTION
**Pull request by mscwilson**
Merging the JavaScript and Browser tracker pages into one Web tracker section. Redirects galore.
The page structure is broadly as before.

Did a bit of general tidying while I was there - removed reusable blocks that were only used once, and gave ones called `Block1234` a proper name.

### CODE REVIEW

**Comment by mscwilson** [comment]
The original sections are still there as previous versions. Perhaps unnecessary?! Might be better to keep just JS v2. These current pages can be moved there once v4 comes out.

**Comment by Jack-Keene** [comment]
It's not in one of the 254 files you have changed, but would it be worth editing [this](https://deploy-preview-637--snowplow-docs.netlify.app/docs/collecting-data/collecting-from-own-applications/javascript-trackers/) page to mention that browser and javascript trackers are found under web trackers?

**Comment by Jack-Keene** [comment]
Again not a section that's been changed, but could we move the setup, initialisation etc pages up a level for [node](https://deploy-preview-637--snowplow-docs.netlify.app/docs/collecting-data/collecting-from-own-applications/javascript-trackers/node-js-tracker/node-js-tracker-v3/) to match the web tracker structure? Not sure it needs a sub header for one tracker version

**Comment by mscwilson** [comment]
First comment: yes good idea
Second comment: also a good idea, but more of a faff (updating links) so I'll bear it in mind for phase 2.

File: static/_redirects [hunk 1]
```diff
@@ -179,3 +179,31 @@ docs/understanding-tracking-design/managing-data-structures-with-data-structures
...
+/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/cookies-and-local-storage/ /docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/cookies-and-local-storage/ 301
+/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/tracker-setup/installing-the-tracker-from-npm/ /docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracker-setup/installing-the-tracker/ 301
+/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/plugins/adding-plugins-to-your-tracker/ /docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/plugins/configuring-tracker-plugins/browser/ 301
+/docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/* /docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/:splat 301 *
```
Comment (stanch) [inline static/_redirects:188]: Why the `*` at the end?

**Pull request merged**